src/components/card/CardSidebar.vue

```diff
@@ -25,6 +25,7 @@
 		:active="tabId"
 		:title="title"
 		:subtitle="subtitle"
+		:subtitleTooltip="subtitleTooltip"
```

Comment:
Mind to fix the eslint warning?

src/components/card/CardSidebar.vue#L28
Attribute ':subtitleTooltip' must be hyphenated